mm-bot/docs/README.md

```diff
@@ -0,0 +1,91 @@
+# MM Bot
+MM Bot is a process designed to supply liquidity to YAB Escrow orders.
+
+## Logical View
+### Functional Requirements
+- The bot must be able to read an order from the Escrow contract.
+- The bot must be able to perform a transfer in Ethereum to the recipient address through the Payment
+Registry contract.
+- The bot must be able to perform a repay in Ethereum to recover the funds in the L2 through the
+Payment Registry contract.
+- The bot must be able to store the orders in a database and update their status.
+- In case of an error, the bot must be able to store the error and retry the order.
+
+### Simplified Class Diagram
+The following diagram shows the mm classes and how they interact with each other.
+
+![mm_diagram_class.svg](images%2Fmm_diagram_class.svg)
+
+### Full Class Diagram
+The following diagram is a detailed version of the previous diagram, 
+showing the attributes and methods of each class.
+
+![mm_diagram_class_full.svg](images%2Fmm_diagram_class_full.svg)
+
+## Process View
+### Non-Functional Requirements
+- The bot must be able to handle multiple orders simultaneously.
+- The bot must be able to retrieve the status of the orders in case of interruption and complete it.
+- The bot must be highly available.
+- The bot must index the orders that belong to accepted blocks to ensure that orders are not lost.
```

Comment:
```suggestion
- The bot must index the orders that belong to accepted blocks to ensure the block being read is final and will not change
```

Reply:
The bot can not revert transactions if the block is reverted